components/common/PanelsCreator.tsx

```diff
@@ -49,6 +50,11 @@ export function PanelsCreator({
                           <Curricula
                             curricula={filteredPanel[panelProp] as Curriculum[]}
                           />
+                        ) : panelProp === "conditionsOfJoining" ? (
+                          <ConditionsOfJoiningField
+                            conditionsOfJoining={filteredPanel[panelProp]}
+                            startDate={filteredPanel["startDate"]}
+                          />
                         ) : (
                           displayListVal(filteredPanel[panelProp], panelProp)
```

Comment:
We can probably leave this nested ternary refactoring until your Coj status and @EdwardBarclay's Coj link are combined into a single Coj comp?